individualmentoring

```diff
@@ -0,0 +1,101 @@
+# How to individually mentor a struggling student
+
+Context:
+
+Many of our students struggle during the JavaScript phase. If you want to mentor them on an individual basis, the folowing approach
+can help you mentor the student in a structured and efficient way. We've cut it up into 3 steps.
+
+##1.Asses
+
+The first step in making a plan for the students you are mentoring would be to assess the level of the student. 
+
+In assessing the skill level, we look for possession of the skillset taught in our curriculum, with as the most important skillset : fundamental knowledge of JavaScript. (and therewith fundamental knowledge of programming)
+
+The following general topics must be understood well with regard to JavaScript:
+
+Basics:
+
+⋅⋅*	Basic Data types [Strings, Numbers, Arrays, Booleans]
+⋅⋅* Operators
+⋅⋅*	Naming conventions
+⋅⋅* Conditions
+⋅⋅* Advanced data types [Objects]
+⋅⋅*	Statements vs Expressions
+⋅⋅* Loops (for/while)
+⋅⋅* Functions
+
+More advanced:
+
+⋅⋅*	Working with the debugger
+⋅⋅* Array Manipulations
+⋅⋅* Map and filter
+⋅⋅* Closures
+⋅⋅* Sound understanding of Scope
+⋅⋅* Callbacks
+⋅⋅* Events
+
+
+Advanced:
+
+⋅⋅* Understanding  of Codeflow
+⋅⋅* Async vs Sync
+⋅⋅* XHTTP requests
+⋅⋅* API calls
+⋅⋅* Rewrite data structures in Json
+⋅⋅* Promises
+
+
+The best way to asses these topics, is 
+
+1. for the students to write code (solving simple programming exercises)(
+2. have them explain their code 
+3. and for you to "checkmark" on the above topics of the curriculum one by one. (which one they know, and which they don't)
+
+We recommend having them write some basic codeassignment, like the ones mentioned below.
+
+Some examples of exercises, based per subject and difficulty. (feel free to add!)
+
+string manipulation: (simple)
+https://www.codewars.com/kata/remove-first-and-last-character/train/javascript
```

Comment:
same comment here, why don't we add these in the corresponding weeks of the curriculum and keep all the content of the program in one place.